policies/project-director-election-process.md

```diff
@@ -0,0 +1,77 @@
+# Project Director Election Process
+
+## Goals
+
+The purpose of this policy is to outline a clear and efficient process for electing new Project Directors to represent the Rust project on the board of the Rust Foundation. The process aims to achieve the following goals:
+
+* Select candidates who fulfill the outlined candidate criteria in the [role description document](TODO: link), focusing on new leadership and diverse perspectives rather than solely prioritizing the best candidate.
```

Comment:
```suggestion
* Select candidates who fulfill the outlined candidate criteria in the [role description document](../roles/rust-foundation-project-director.md), focusing on new leadership and diverse perspectives rather than solely prioritizing the best candidate.
```